src/transformer/decode.ts

```diff
@@ -150,6 +152,9 @@ const getAffiliations = (modelMap: Map<string, Model>) =>
 const getContributors = (modelMap: Map<string, Model>) =>
   getModelsByType<Affiliation>(modelMap, ObjectTypes.Contributor)
 
+const getTitle = (modelMap: Map<string, Model>) =>
+  getModelsByType<Title>(modelMap, ObjectTypes.Title)[0]
```

Comment:
you could replace this with `findTitle` or use `getModelsByType(model).find...`, as assuming there will always be a MPTitle could break decoding if it's missing

Reply:
I'm doing a check on line line 825.. but I will replace it with findTitle
![image](https://github.com/Atypon-OpenSource/manuscripts-transform/assets/93189633/a5bbb397-b7e9-4284-b54b-b1a1c0057991)

Reply:
I'm doing a check on line 825